package.json

```diff
@@ -13,9 +13,13 @@
     "url": "https://github.com/AnomalyInnovations/serverless-nodejs-starter.git"
   },
   "devDependencies": {
+    "@shelf/jest-mongodb": "^1.1.5",
     "aws-sdk": "^2.636.0",
+    "jest": "^25.2.3",
+    "mongodb-memory-server": "^6.3.3",
     "serverless-bundle": "^1.2.5",
-    "serverless-offline": "^5.3.3"
+    "serverless-offline": "^5.3.3",
+    "supertest": "^4.0.2"
```

Comment:
are we using all these?

I can only see where supertest is used and `@shelf/jest-mongodb` is being setup (and jest of course)... mongodb-memory-server and serverless-offline seem a bit invisible to me right now...

Reply:
they will be implemented later on for testing POST routes.

Reply:
Now that I think about it, will also need to figure out a way to mock a jwt token to get to those routes or find a way to test those lambdas minus the authorizer function.

Reply:
probably mocking out the entire cognito client might be the only way.